project/recursive_sorting.py

```diff
@@ -24,8 +29,8 @@ def merge( arrA, arrB ):
 ### recursive sorting function
 def merge_sort( arr ):
     if len( arr ) > 1:
-        left = merge_sort( arr[ 0 : len( arr ) / 2 ] )
-        right = merge_sort( arr[ len( arr ) / 2 : ] )
+        left = merge_sort( arr[0:int(len(arr) / 2)] )
```

Comment:
I'm not sure if I've mentioned this to you, but another way (to avoid using int() ) to get the division to an integer is to use `//`
8/2 = 4.0
8//2 = 4
9/2 = 4.2
9//2 = 4